agent_build/requirement-files/testing-requirements.txt

```diff
@@ -40,3 +40,5 @@ futures==3.3.0; python_version <= '2.7'
 more-itertools==8.5.0; python_version >= '3.0'
 more-itertools==5.0.0; python_version <= '2.7'
 distro==1.6.0
+xmltodict==0.13.0; python_version >= '3.0'
+xmltodict==0.12.0; python_version <= '2.7'
```

Comment:
pre-built Windows Python binaries are Python 3 only so we can remove this dependency for <= 2.7 and update unit tests so they only run under Python 3 (aka skip tests which rely on this dependency under Python 2).

Reply:
Will fix